example/RotatedSC_d3.jl

```diff
@@ -228,6 +228,7 @@ function check_surface_code_decoder(d::Integer)
         # for cfg in cfgs1
```

Comment:
If I do the below change to be consistent across the board, I get the following error "The Stabilizer does not match, the program is wrong even without error insertion" irrespective of the fact that I am using a correct/incoreect decoder.
```suggestion
        for cfg in cfgs1
            if !check_state_equivalence(
                  # ρ01, ρ01, (ϕ_x1 #=& ϕ_z1=#, cfg1.ϕ[1], cfg1.ϕ[2]),
                   cfg.ρ, ρ01, (ϕ_x1 #=& ϕ_z1=#, cfg.ϕ[1], cfg.ϕ[2]),
                   `bitwuzla --smt-comp-mode true -rwl 0 -S kissat`
                    #`bitwuzla --smt-comp-mode true -S kissat`
                     #`bitwuzla --smt-comp-mode true -rwl 0`
                    )
                  res = false
                 break
                 end
        end
```